data/locales/si.lua

```diff
@@ -0,0 +1,86 @@
+return {
+-- Buttons, context menu
+['ui_use'] = "Uporaba",
+['ui_give'] = "Daj",
+['ui_close'] = "Zapri",
+['ui_drop'] = "Spusti",
+['ui_removeattachments'] = "Odstrani priloge",
+-- Tooltip
+['ui_durability'] = "Trajnost",
+['ui_ammo'] = "strelivo",
```

Comment:
Should this be lowercase?

Reply:
yes

Reply:
in our language we would type that in lowercase

Reply:
I have no idea how that makes any sense, but cool!

Reply:
ui_removeattachments is not right also